### ISSUE TITLE
Hidive No longer muting.

### ISSUE THREAD

**Issue by Andycodesforyou** [FrostCo/AdvancedProfanityFilter#548]
####  Bug Description
Hello Hidive on multiple devices no longer mutes filters words but the filtering in the captions still work (words are still blotted out). Hidive recently changed their site and updated/changed their video player so I would hazard a guess that this has something to do with it. 

####  Steps To Reproduce
1. Go to Advance profanity filter options
2. Click on Audio  --> mute audio on supported video sites
3. Go to Hidive, play a video
4.  Audio not muting on Hidive videos

#### Expected behavior

Audio muting of videos playing

#### Devices tested
Macbook pro (2016, Monterey) on Chrome Browser.
Samsung Tablet (Android) on Kiwi browser
Blackview phone (Android) on Kiwi browser.

#### Settings tested
On the Black view phone tested all the various mute settings. 
For Macbook and Samsung only tested standard settings but can test the other settings if needed (assuming it is the same as phone). 

Both Youtube's and Crunchyroll's muting works on these other devices so it seems to be specifically Hidive.

**Comment by richardfrost** [comment]
Hi @Andycodesforyou,

Thanks for reporting the issue! It looks like they have updated/changed their website quite a bit, which is preventing the filter from working. Could you please give this config a try and see if it works for you? If it doesn't work, please send me a specific video or two where it isn't working so I can get it fixed.

```json
{
  "www.hidive.com": [
    {
      "className": "ds-text-track__wrapper-container",
      "displaySelector": "div.ds-text-track",
      "mode": "element",
      "subtitleSelector": "span.ds-text-track__text",
      "tagName": "DIV"
    }
  ]
}
```

To use this config, paste it into the Custom Sites area (Options page > Audio tab) and click the blue "SAVE" button underneath it.

**Comment by Andycodesforyou** [comment]
Okay I will do that. Sorry for late replay.

**Comment by Andycodesforyou** [comment]
Tested this on Chrome. It works. Will test on the other devices and get back to you @richardfrost

**Comment by Andycodesforyou** [comment]
Tested on both Android phone and Samsung tablet in Kiwi browser. 

The config file works. @richardfrost .

**Comment by richardfrost** [comment]
Thanks @Andycodesforyou for testing it out and getting back to me. I'll add that new config to the next version of the Extension.

**Comment by richardfrost** [comment]
This has been included in version [3.5.0](https://github.com/FrostCo/AdvancedProfanityFilter/releases/tag/3.5.0) of the extension. For best results, make sure to remove any custom config used for testing like this once it has been included in the official update so you can automatically get future updates.
